tests/tests/test-xcm/test-xcmv3-new-instructions.ts

```diff
@@ -0,0 +1,382 @@
+      ).index;
+
+      // Send some native tokens to the sovereign account of paraId (to pay fees)
+      const { result } = await context.createBlock(
+        createTransaction(context, {
+          ...ALITH_TRANSACTION_TEMPLATE,
+          value: 1_000_000_000_000_000_000,
+          to: paraSovereign,
+          data: "0x",
+        })
+      );
+      expectEVMResult(result.events, "Succeed");
+
+      amount = 1_000_000_000_000_000n;
+      dotAsset = {
+        assets: [
+          {
+            multilocation: {
+              parents: 0,
+              interior: {
+                X1: { PalletInstance: balancesPalletIndex },
+              },
+            },
+            fungible: amount,
+          },
+        ],
+        // weight_limit: new BN(4000000000),
+        beneficiary: CHARLETH_ADDRESS,
+      };
+    });
+
+    it("Should execute BurnAsset", async function () {
+      const xcmMessage = new XcmFragment(dotAsset)
+        .withdraw_asset()
+        .buy_execution()
+        .burn_asset()
+        .as_v3();
+
+      // Mock the reception of the xcm message
+      await injectHrmpMessage(context, paraId, {
+        type: "XcmVersionedXcm",
+        payload: xcmMessage,
+      } as RawXcmMessage);
+      await context.createBlock();
+
+      // Search for Success
+      const records = (await context.polkadotApi.query.system.events()) as any;
+      const events = records.filter(
+        ({ event }) => event.section == "xcmpQueue" && event.method == "Success"
+      );
+      expect(events).to.have.lengthOf(1);
+    });
+
+    it("Should execute ClearTopic", async function () {
+      const xcmMessage = new XcmFragment(dotAsset)
+        .withdraw_asset()
+        .buy_execution()
+        .clear_topic()
+        .as_v3();
+
+      // Mock the reception of the xcm message
+      await injectHrmpMessage(context, paraId, {
+        type: "XcmVersionedXcm",
+        payload: xcmMessage,
+      } as RawXcmMessage);
+      await context.createBlock();
+
+      // Search for Success
+      const records = (await context.polkadotApi.query.system.events()) as any;
+      const events = records.filter(
+        ({ event }) => event.section == "xcmpQueue" && event.method == "Success"
+      );
+      expect(events).to.have.lengthOf(1);
+    });
+
+    it("Should execute ExpectTransactStatus", async function () {
+      const xcmMessage = new XcmFragment(dotAsset)
+        .withdraw_asset()
+        .buy_execution()
+        .expect_transact_status()
+        .as_v3();
+
+      // Mock the reception of the xcm message
+      await injectHrmpMessage(context, paraId, {
+        type: "XcmVersionedXcm",
+        payload: xcmMessage,
+      } as RawXcmMessage);
+      await context.createBlock();
+
+      // Search for Success
+      const records = (await context.polkadotApi.query.system.events()) as any;
+      const events = records.filter(
+        ({ event }) => event.section == "xcmpQueue" && event.method == "Success"
```

Comment:
same comment as in the `tests/tests/test-xcm/test-xcmv3-max-weight-instructions.ts`